packages/fields-cloudinary-image/src/test-fixtures.js

```diff
@@ -1,6 +1,247 @@
+const path = require('path');
+require('dotenv').config({ path: path.resolve(__dirname, '../.env') });
```

Comment:
What's the purpose of this line?

Reply:
@timleslie : This is to grab all the env variables defined in  `.env` file. 
Additionally, `{path: ...}` was required to identify the nested `.env` file location as node process was not picking up if we leave the config option object empty.